website/src/layouts/base/header/Brand.tsx

```diff
@@ -1,10 +1,29 @@
-export function Brand({ name }: { name: string }) {
+export function Brand() {
     return (
-        <div>
-            <a href='/' className='my-8 flex items-center'>
-                <img className='ml-4 mr-2' src='/favicon.png' alt='icon' />
-                <div className='ml-2 font-bold'>{name}</div>
+        <div className='my-8 flex items-center'>
+            <a href='/'>
+                <img className='ml-4 mr-2' src='/virus-outline-colorful.svg' alt='icon' width='20' height='20' />
             </a>
+            <BrandName />
         </div>
     );
 }
+
+type BrandNameProps = {
+    fontsize?: string;
+};
+
+export function BrandName({ fontsize = 'font-bold' }: BrandNameProps) {
+    return (
+        <a href='/' className='ml-2'>
```

Comment:
Should we remove the `a` here? 
* In `Brand`, we can simply put the `BrandName` inside the other `a` that is already there.
* On the `index.astro` landing page, IMO we don't need the link at all.